use_cases/gretel.json

```diff
@@ -6,7 +6,7 @@
       "description": "Create highly accurate synthetic training data for ML models.",
       "cardType": "Console",
       "imageName": "synthetics.png",
-      "icon": "synthetics.png",
```

Comment:
**important:** we need the icon here, as we are transitioning from using the `imageName` field to `icon`. I'm going to remove the `imageName` and card header images in the folder in another week or so.

Reply:
ah thanks, i didn't catch that. Good to know.